Game/buildings.gd

```diff
@@ -0,0 +1,21 @@
+extends Node
+
+@export var _buildings_dict: Dictionary[Types.Building, BuildingResource]
```

Comment:
We definitely need this somewhere, but I feel like it shouldn't be here. Though where it should be I'm not sure. Okay to leave here for now.

Also, who is populating this?  We should probably auto-populate it at startup by looking through the list of buildings and processing them.